src/alarms/notifications/ErrorOverlay.js

```diff
@@ -0,0 +1,216 @@
+import CSSTransition from "react-transition-group/CSSTransition";
+import MDSpinner from "react-md-spinner";
+import React, { Component } from "react";
+import styles from "../../components/ErrorOverlay.css";
+import Lottie from "react-lottie";
+import * as animationSucces from "../../data_management/rasters/success.json";
+import * as animationError from "../../data_management/rasters/error.json";
+import buttonStyles from "../../styles/Buttons.css";
+import { FormattedMessage } from "react-intl";
+
+// Based on the OrganisationSwitcher, this overlay gets called when an error occurs during the upload process.
+class ErrorOverlay extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      width: window.innerWidth,
+      height: window.innerHeight,
+      isStopped: false,
+      isPaused: false
+    };
+    this.handleResize = this.handleResize.bind(this);
+    this.hideErrorOverlay = this.hideErrorOverlay.bind(this);
+    this.whichAnimation = this.whichAnimation.bind(this);
+    this.whichMessage = this.whichMessage.bind(this);
+    this.succesButtons = this.succesButtons.bind(this);
+  }
+  componentDidMount() {
+    window.addEventListener("resize", this.handleResize, false);
+    document.addEventListener("keydown", this.hideErrorOverlay, false);
+  }
+  componentWillUnmount() {
+    window.removeEventListener("resize", this.handleResize, false);
+    document.removeEventListener("keydown", this.hideErrorOverlay, false);
+  }
+
+  // Resize the overlay
+  handleResize() {
+    this.setState({
+      width: window.innerWidth,
+      height: window.innerHeight
+    });
+  }
+
+  // Close the overlay
+  hideErrorOverlay(e) {
+    if (e.key === "Escape") {
+      this.props.handleClose();
+    }
+  }
+
+  // Animation based on success or fail of a request
+  whichAnimation() {
+    var animationData;
+    if (
+      this.props.errorMessage.status === 201 ||
+      this.props.errorMessage.status === 200
+    ) {
+      animationData = animationSucces.default;
+    } else {
+      animationData = animationError.default;
+    }
+    return animationData;
+  }
+
+  // Different buttons based on succes or fail of a request
+  succesButtons() {
```

Comment:
I think "showSuccessButtons" is a better name for a functioning returning a boolean; with this name it sounds like it returns the buttons themselves.